atlp-rwanda/winners-c8-bn-be
# Fix users endpoint

#### What does this PR do?
1.  Fetch all users in database(authenticated for only admin)
   
#### How should this be manually tested?
1. Clone this repository
2. run `npm install` to bring modules
3. install .env file as shown in .env.example
4. run `npm run dev` to start sever
5. check users endpoint(only allowed for admin)
